compiler/optimizer/OMRValuePropagation.hpp

```diff
@@ -600,10 +600,19 @@ class ValuePropagation : public TR::Optimization
          : _clazz(clazz), _isFixed(isFixed)  { }
    };
 
+   struct ArrayCloneInfo {
```

Comment:
Is this needed? It's exactly the same as `ObjCloneInfo`

Reply:
I thought since the related code seems to have a naming convention in distinction between "object" and "array", like `transformObjectCloneCall` and `transformArrayCloneCall`, adding the `ArrayCloneInfo` struct would lend bit more clarity and consistency. But yes, the structs are the currently same.